etc/kayobe/ansible/smartmon-tools.yml

```diff
@@ -0,0 +1,43 @@
+---
+- hosts: overcloud
+
+  tasks:
+    - name: Ensure smartmon-tools and nvme-cli is installed
+      package:
+        name:
+          - smartmontools
+          - nvme-cli
+          - jq
+        state: present
+      become: true
+
+    - name: Copy smartmon.sh and nvmemon.sh from scripts folder
+      copy:
+        src: "scripts/{{ item }}"
+        dest: /usr/local/bin/
+        owner: 'root'
+        group: 'root'
+        mode: '0700'
+      loop:
+        - smartmon.sh
+        - nvmemon.sh
+      become: yes
+
+    - name: Set PATH Variable for cron
+      cron:
+        name: PATH
+        user: root
+        env: yes
+        job: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+      become: yes
+
+    - name: Schedule cronjob to run both scripts every 5 minutes and save output to file
+      cron:
+        name: "SMART metrics for drive monitoring using {{ item }}"
+        user: root
+        minute: "*/5"
+        job: "/usr/local/bin/{{ item }}.sh > /var/lib/docker/volumes/textfile/_data/{{ item }}.prom.temp && mv /var/lib/docker/volumes/textfile/_data/{{ item }}.prom.temp /var/lib/docker/volumes/textfile/_data/{{ item }}.prom"
```

Comment:
It looks like there's some kolla-ansible configuration required to consume this output? Could you write up how to use this feature? It can be included in the documentation once it is merged: https://github.com/stackhpc/stackhpc-kayobe-config/pull/252/files

Reply:
Yep I am currently in the process of writing it up. Would you prefer that I include the relevant changes (like creating the docker volume and enabling text collector for node exporter) as instructions in the wiki or would you rather that I just include those changes to the config files in this PR (since creating a new docker volume doesn't really consume any space). 

Let me know what you think would work best!

Reply:
I think we could enable the textfile collector out of the box, and then document the custom playbook to write metrics to that folder?